css/src/components/segmented-control.scss

```diff
@@ -0,0 +1,131 @@
+$sc-padding-sm: 0.5em 1em !default;
+$sc-padding: 0.75em 1.25em !default;
+$sc-padding-lg: 1em 1.5em !default;
+
+$sc-font-color: $primary !default;
+$sc-border: $border-width solid $border !default;
+$sc-hover-background-color: $primary-background !default;
+$sc-hover-border: $border-width solid $border !default;
+$sc-hover-border-color: $primary-hover !default;
+$sc-active-hover-background-color: $primary-hover !default;
+$sc-active-hover-border-color: $primary-hover !default;
+$sc-active-background-color: $primary !default;
+$sc-active-border-color: $primary !default;
+$sc-active-color: $text-invert !default;
+
+.segmented-controls {
+	display: grid;
+	justify-content: center;
+	width: 100%;
+	grid-gap: $spacer-5;
+	grid-template-columns: max-content auto max-content;
+	font-size: $font-size-7;
+	white-space: nowrap;
+
+	&.segmented-lg {
+		font-size: $font-size-5;
+	}
+}
+
+.segmented-control-previous,
+.segmented-control-next {
+	display: flex;
```

Comment:
are we going to support hiding these buttons for certain use-cases?

Reply:
I will be exploring our options on it today.

Reply:
It's a good question. I'd say if either of the nav buttons are hidden, we'll have a bit of an inconsistency between `tab-container`'s built-in keyboard behavior and what is accomplishable by the buttons. I'm not sure how much we'd really want that.

Reply:
So I just tested against this without the next/prev buttons and the left/right action on arrow keys worked great.  But still more to be checked.